framework/src/NormalisedSumPDF.cpp

```diff
@@ -248,12 +245,24 @@ void NormalisedSumPDF::MakePrototypes( PhaseSpaceBoundary * InputBoundary )
 			}
 		}
 	}
+	// Make sure the cached integrals have the right indices. Set everything to 1 to avoid division by zero. Old cache is invalid anyway.
```

Comment:
I'm not sure I understand why this is being called here? Is this because it fixes things in the component plotting integrations? If so then I think the component plotting should make a copy of the PDF if there are problems as the internal state of the PDF isn't guaranteed to be consistent with what is being done there.